tests/ops/qubit/test_parametric_ops.py

```diff
@@ -2015,6 +2015,7 @@ def test_globalphase_autograd_grad(self, tol, dev_name, diff_method, wires):
         @qml.qnode(dev, diff_method=diff_method)
         def circuit(x):
             qml.Identity(wires[0])
+            qml.GlobalPhase(x, wires=[0, 1])  # Does not change the derivative, but tests it
```

Comment:
Test `GlobalPhase` derivative directly, without drawing in `AmplitudeEmbedding`.